### PR TITLE
Remove elasticsearch, increase graf timeout

### PR DESCRIPTION
**Pull request by mckornfield**
* Also edit setup script

### CODE REVIEW

File: monitoring/roles/loki/files/values.yaml [hunk 1]
```diff
@@ -38,7 +38,7 @@ loki:
     enabled: true
     storageClassName: do-block-storage
     accessModes: ["ReadWriteOnce"]
-    size: 50Gi
+    size: 100Gi
```
Comment (mckornfield) [inline monitoring/roles/loki/files/values.yaml:41]: I made this live for prod, but didn't backport to lower envs. I will keep an eye on the lower ones and see if I need to backport it. Also wondering if we'll need more memory for loki/ run it distributed

Comment (TANguyen1893) [reply monitoring/roles/loki/files/values.yaml:41]: Does distributed mean HA? I think it would be good to be HA, especially for monitoring.

Comment (mckornfield) [reply monitoring/roles/loki/files/values.yaml:41]: Yep, though it is more for the query side than anything

Comment (mckornfield) [reply monitoring/roles/loki/files/values.yaml:41]: The log scraping is a daemonset

Comment (mckornfield) [reply monitoring/roles/loki/files/values.yaml:41]: I can make things HA in a follow up if there's a need

**Pull request merged**